### ISSUE TITLE
2.13.1

### ISSUE THREAD

**Issue by khaoua** [nickian/mautic-whitelabeler#24]
Hey Nick

quick one to know if 2.13.1 compatibility is effective or not ? did not upgrade yet ;-)

**Comment by krabhishek** [comment]
Tested it on 2.13.1 and it works fine. Ensure to edit the version.txt file to 2.12.2 before starting the whitelabeling process. You can change the same back to 2.13.1 once the run is complete.

**Comment by khaoua** [comment]
great news @krabhishek thanks, will test that shortly, thanks ;-)

**Comment by imrodmartin** [comment]
hmmm
I'm getting constant redirects with the final error "To many redirects"...
permissions/ownership is correct...
any thoughts?

**Comment by khaoua** [comment]
haven't faced that issue, but indeed, most probably a server issue, which one, I wish I could help, but no idea at the moment. For the site that worked, did you follow the process described by krabhishek

**Comment by imrodmartin** [comment]
yep... same process for both.

**Comment by krabhishek** [comment]
Yes, it’s because of permission issues as well as most likely because of
.htaccess file. I had faced the same issue.

Are you using bitnami images by any chance? If so, please ensure you run
chown -R root:daemon on the folder with whitelabeler code and try. It
should be fixed and the code should run just fine.

**Comment by khaoua** [comment]
ok then I guess it should work, or make sure your host has proper logs on what happened during whitelabeling

**Comment by imrodmartin** [comment]
ok - so i uploaded the older version of whitelabler  (downloaded in January) - and it's working... 
(and no - not using bitnami)

**Comment by khaoua** [comment]
kinda weird, but only result counts 👍

**Comment by imrodmartin** [comment]
yep! thanks for going back and forth with me

**Comment by khaoua** [comment]
with pleasure

**Comment by khaoua** [comment]
Hey @nickian , thanks for the feedback, will go ahead for the updgrade and whitelabel again, if you say it works with 2.13.1, your contribution has always been valuable, so thanks again for the sharing

**Comment by khaoua** [comment]
@nickian works like a charm, ;-)
